gradle/publishing.gradle

```diff
@@ -29,8 +29,6 @@ if (!isBom) {
 task emptyJar(type: Jar) {
 }
 
-def bintrayUploadFlag = System.getenv("libs.bintray.upload") != null
```

Comment:
Perhaps it is worth updating the comment `Configures publishing of Maven artifacts to Bintray`